happy/HappyInternet.py

```diff
@@ -324,6 +325,8 @@ def __internet_state(self):
             isp_dic["isp_addr"] = self.isp_addr
             isp_dic["node_id"] = self.node_id
             isp_dic["isp_index"] = self.isp_index
+            isp_dic["isp"] = self.isp_id
```

Comment:
Lines 328 and 329 do not appear to be used anywhere. What do they do? What should they do?

Reply:
this will be write to a .happy_state.json file and will be used during happy-internet delete, which loads the .happy_state.json and get values from there.